src/lib/candid_types/index.ts

```diff
@@ -5,43 +5,186 @@ export { Func } from './func';
 export { Service } from './service';
 export { Variant } from './variant';
 
+/** Marks a type as being an alias to a candid type */
 export type Alias<T> = T;
+
+/**
+ * Represents a sequence of bytes. Synonymous with `Vec<nat8>`.
```

Comment:
We need to check on `Vec<nat8>`, I'm not sure that will work. Even if it works technically, the TypeScript types would be incorrect. I don't think it's a best practice to share with people, and we should probably discourage and thus not mention it.